kriswallsmith/buzz/1.0/config/packages/buzz.yaml

```diff
@@ -0,0 +1,54 @@
+# Read more about client options at: https://github.com/kriswallsmith/Buzz/blob/master/doc/client.md
+services:
+    _defaults:
+        public: false
+
+    # Tell the auto wire system to use "FileGetContents" client as default client.
+    # Set this to any of the other clients below (e.g. Curl or MultiCurl) to use a different default client.
+    Buzz\Client\BuzzClientInterface: '@buzz.client.file_get_contents'
+    Buzz\Client\BatchClientInterface: '@buzz.client.multi_curl'
+
+    # Register alias for PSR-18
+    Psr\Http\Client\ClientInterface: '@Buzz\Client\BuzzClientInterface'
+
+    # Clients
+    buzz.client.file_get_contents:
```

Comment:
Why not using the fqcn ?

Reply:
In this case Most of class: are obsolete too and would decrease the config file size too

Reply:
@nyholm looks like you miss this comments (oh I merged this pr by approving I guess)

Reply:
Yes, you did. 

I did not use FQCN on these because they should not really be "public". You should type hint to the interface not the concrete implementation. So the only time you should use this service id is when you write your alias.